### PR TITLE
Feature/store version and converter

### PR DESCRIPTION
**Pull request by jcwinkler**
A tiny pr - to warm up on github :-)

The hard coded strings "pdflatex" and "pstoedit" will be replaced later by class methods returning the required information (see detailed commit message).

### CODE REVIEW

**Comment by leberwurstsaft** [comment]
Very good idea. I'd argue to rename "textextver" to "version", since the namespace is already in there.

File: textext.py [hunk 1]
```diff
@@ -307,6 +307,9 @@ def do_convert(self, text, preamble_file, scale_factor, converter_class, old_nod
             return
 
         # -- Store textext attributes
+        new_node.attrib['{%s}textextver' % TEXTEXT_NS] = __version__.encode('string-escape')
```
Comment (leberwurstsaft) [inline textext.py]: Call it "version" instead of "textextver". The namespace is already about textext.

Comment (leberwurstsaft) [reply textext.py]: The idea to store these attributes is great, btw.

Comment (jcwinkler) [reply textext.py]: Right. Changed it.

**Pull request merged**